tox.ini

```diff
@@ -21,7 +21,7 @@ deps =
     django22: Django==2.2.*
     django31: Django==3.1.*
     django32: Django==3.2.*
-    djangomain: https://github.com/django/django/archive/main.tar.gz
+    ; djangomain: https://github.com/django/django/archive/main.tar.gz
```

Comment:
Current main branch of Django has changed behavior for for labels. Not sure if this is intentional. We might need to file a bug report.